odoo/fields.py

```diff
@@ -992,7 +992,7 @@ def __get__(self, record, owner):
                     ]))
                 value = env.cache.get(record, self)
 
-            elif self.compute:
```

Comment:
wasn't it `and not self.readonly`?

Reply:
Yes, but it was making many other tests crash.
This patch seems to break less things...

Reply:
"less"